index.html

```diff
@@ -337,11 +337,28 @@ <h1 class="skills-2">
                 </h1>
                 <hr>
             </div>
-
+             
 
         </div>
 
     </section>
+    <!-- -----------------------------------------FOOTER CONTACT FORM ----------------------------------------------- -->
+            <section class="contact-form">
+                <div class="contact-global-container">
+                 <h2 class="contact-text-container">Contact me </h2>
+                 <p class="contact-paragraph">If you have an application  you <br>are interested in developing, a <br> feature that you need built or a <br> project that needs coding.
+                    I’d <br> love to help with it</p>
+                    <form action="https://formspree.io/f/xzbwnnow" class="submit-form" method="post">
+                     <input type="text" name="name" class="username" placeholder="Yeremias" minlength="30" required="required"> 
+                    <input type="email" name="email" name="replyto" class="email" placeholder="natajayanj14@gmail.com" required="required">
+                     <textarea name="text-field" class="text-area" cols="30" rows="10" minlength="500" placeholder="Write your message here" required="required"></textarea>
```

Comment:
- [ ] Your form works but the name and message inputs should have a **maximum** of 30 and 500 characters, not a **minimum**. Kindly fix this issue to meet the requirements. 👍

Reply:
Thank you I have changed them